api/src/test/java/com/staxpayments/api/datasource/InvoiceFakeRepository.kt

```diff
@@ -0,0 +1,73 @@
+package com.staxpayments.api.datasource
+
+import com.staxpayments.api.models.Invoice
+import com.staxpayments.api.repository.InvoiceRepository
+import com.staxpayments.api.requests.CreateInvoiceBody
+import java.util.*
+import kotlin.collections.LinkedHashMap
+
+class InvoiceFakeRepository: InvoiceRepository {
```

Comment:
we don't need this class to test..

we just need to create a data class and pass the data class as mock... we have a sample test case in Android code domain layer part, I can show you if you can't get it

Reply:
I agree with Muraino. If you need help, sync with him on the Android tests he's written before